ui/src/api-logic/graphql/queries/get-repos.ts

```diff
@@ -65,7 +65,7 @@ const GET_REPOS = gql`
           vendor
           settings
         }
-        repoSyncs(orderBy: SYNC_TYPE_ASC) {
+        repoSyncs(orderBy: SYNC_TYPE_ASC, condition:{scheduleEnabled:true}) {
```

Comment:
```suggestion
        repoSyncs(
          orderBy: SYNC_TYPE_ASC
          filter: {lastCompletedRepoSyncQueueId: {isNull: false}}
        ) {
```

Reply:
This way it works... and run `npm run codegen` to update generated schema on the UI (_/ui/src/api-logic/graphql/generated/schema.ts_).

Reply:
done